QRCoder/QRCodeGenerator.cs

```diff
@@ -1369,8 +1352,10 @@ private static int GetIntValFromAlphaExp(int exp)
         private static int GetAlphaExpFromIntVal(int intVal)
         {
             if (intVal == 0)
-                throw new ArgumentOutOfRangeException(nameof(intVal));
+                ThrowArgumentOutOfRangeException(nameof(intVal));
             return galoisFieldByIntegerValue[intVal];
+
+            void ThrowArgumentOutOfRangeException(string paramName) => throw new ArgumentOutOfRangeException(paramName);
```

Comment:
```suggestion
            void ThrowIntValIs0() => throw new ArgumentOutOfRangeException(nameof(intVal));
```
?
(similar to above)

Reply:
Done